resources/rafter/charts/controller-manager/templates/assetgroups.crd.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if .Values.installCRDs -}}
+{{- if not (.Capabilities.APIVersions.Has "rafter.kyma-project.io/v1beta1/AssetGroup") }}
+{{- if .Release.IsUpgrade }}
```

Comment:
Anyway, it should be rather `or`? If the cluster already has `rafter.kyma-project.io/v1beta1/AssetGroup` then it will not upgrade, because the first `if` will be false.

Reply:
But hm... it will be still wrong. The first installation will skip that, but the next one will update to the version that is in a chart...

Reply:
As agreed
- we will remove rafter `CRDs` from `kyma/rafter` chart
- we will leave rafter `CRDs` in `cluster-essentials` 
- we will create separate issue to install `assets`, `clusterassetgroups`, `assetgroups`, `clusterbuckets`, `buckets`, `clusterassets` in a `Job` after we migrate to `helm3`